lib/collection/header.js

```diff
@@ -105,13 +105,13 @@ _.assign(Header.prototype, /** @lends Header.prototype */ {
          * @type {String}
          * @todo avoid headers with falsy key.
          */
-        this.key = _.get(options, 'key') || E;
+        this.key = _.isNil(_.get(options, 'key')) ? E : options.key;
```

Comment:
Same as:

```suggestion
        this.key = _.get(options, 'key', E);
```

Reply:
No, it's different.
`_.get()` will not return default value for `null`.